frontend/pages/policies/PolicyPage/PolicyPage.tsx

```diff
@@ -122,22 +116,17 @@ const PolicyPage = ({
     }
   );
 
-  // if URL is like `/policies/1?host_ids=22`, add the host
-  // to the selected targets automatically
   useQuery<IHostResponse, Error, IHost>(
     "hostFromURL",
     () =>
       hostAPI.loadHostDetails(parseInt(URLQuerySearch.host_ids as string, 10)),
     {
       enabled: !!URLQuerySearch.host_ids,
       select: (data: IHostResponse) => data.host,
-      onSuccess: (data) => {
+      onSuccess: (host) => {
         const targets = selectedTargets;
-        const hostTarget = data as any; // intentional so we can add to the object
-
-        hostTarget.target_type = "hosts";
-
-        targets.push(hostTarget as IHost);
```

Comment:
Correct approach here is to add the prop to the necessary interface. This approach bypasses the benefit of TypeScript.